### ISSUE TITLE
[BUG] COMPODOC_SEARCH_INDEX is not defined

### ISSUE THREAD

**Issue by andrew19881123** [compodoc/compodoc#377]
##### **Overview of the issue**
```javascript
Uncaught ReferenceError: COMPODOC_SEARCH_INDEX is not defined
    at LunrSearchEngine.init (search-lunr.js:13)
    at Object.<anonymous> (search-lunr.js:59)
    at Object.dispatchEvent (EventDispatcher.js:88)
    at HTMLDocument.<anonymous> (compodoc.js:11)
LunrSearchEngine.init	@	search-lunr.js:13
(anonymous)	@	search-lunr.js:59
dispatchEvent	@	EventDispatcher.js:88
(anonymous)	@	compodoc.js:11
```
##### **Operating System, Node.js, npm, compodoc version(s)**

angular/cli: 1.4.5
node: 6.11.0
os: win32 x64
compodoc: 1.0.3

##### **Angular configuration, a `package.json` file in the root folder**
```json
"devDependencies": {
    "angular/animations": "4.4.4",
    "angular/cli": "1.4.5",
    "angular/common": "4.4.4",
    "angular/compiler": "4.4.4",
    "angular/compiler-cli": "4.4.4",
    "angular/core": "4.4.4",
    "angular/forms": "4.4.4",
    "angular/http": "4.4.4",
    "angular/language-service": "4.4.4",
    "angular/platform-browser": "4.4.4",
    "angular/platform-browser-dynamic": "4.4.4",
    "angular/router": "4.4.4",
    "compodoc/compodoc": "1.0.3",
    "types/chai-as-promised": "^7.1.0",
    "types/jasmine": "~2.5.54",
    "types/jasminewd2": "2.0.3",
    "types/node": "6.0.89",
    "bootstrap": "4.0.0-beta",
    "chai": "4.1.2",
    "chai-as-promised": "7.1.1",
    "codelyzer": "~3.1.1",
    "core-js": "2.5.1",
    "cucumber": "2.2.0",
    "cucumber-tsflow": "2.2.0",
    "font-awesome": "4.7.0",
    "jasmine-core": "~2.6.2",
    "jasmine-spec-reporter": "~4.1.0",
    "jquery": "3.2.1",
    "karma": "1.7.1",
    "karma-chrome-launcher": "~2.1.1",
    "karma-cli": "~1.0.1",
    "karma-coverage-istanbul-reporter": "1.2.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "0.2.2",
    "popper.js": "1.12.5",
    "primeng": "4.2.2",
    "protractor": "~4.0.0",
    "protractor-cucumber-framework": "4.0.8",
    "rxjs": "5.4.3",
    "ts-node": "~3.2.0",
    "tslint": "~5.3.2",
    "typescript": "~2.3.4",
    "zone.js": "0.8.18"
  },
```

**Comment by vogloblinsky** [comment]
Duplicate of https://github.com/compodoc/compodoc/issues/326

**Comment by vogloblinsky** [comment]
@andrew19881123 
Could you describe a little more the context of your issue ?
Did Compodoc breaks ?
Can you test last version 1.0.4 ?

**Comment by andrew19881123** [comment]
I have tested compodoc 1.0.4 but i get the same result.
Yes, compodocs breaks, as described in the issue [378](https://github.com/compodoc/compodoc/issues/378), but i think the errors are not related to each other.. i had this error since version 1.0.3, and the other from 1.0.0

**Comment by vogloblinsky** [comment]
Yes your issue is the result of https://github.com/compodoc/compodoc/issues/378
We have to fix first 378, this one will i hope be fixed too.

**Comment by vogloblinsky** [comment]
Issue fixed like #378 & #326.
Release coming today for that.
